### ISSUE TITLE
What is Pogress.png file?

### ISSUE THREAD

**Issue by orouskhani** [MIC-DKFZ/nnUNet#1213]
Hello Fabian, 

About the result folders, I mean fold_0, fold_1, ... which provides the training results. There is an image file "progress.png" where the horizontal axis is the number of iteration and the vertical axis is the evaluation metric. 
Could  you pls let me know what is the evaluation metric in this image file? 

Thank you for your help. 
Regards,
Maysam

**Comment by evandrosks** [comment]
Hello Maysam,

as per the [readme](https://github.com/MIC-DKFZ/nnUNet/blob/master/readme.md)

> progress.png: A plot of the training (blue) and validation (red) loss during training. Also shows an approximation of the evlauation metric (green). This approximation is the average Dice score of the foreground classes. It should, however, only to be taken with a grain of salt because it is computed on randomly drawn patches from the validation data at the end of each epoch, and the aggregation of TP, FP and FN for the Dice computation treats the patches as if they all originate from the same volume ('global Dice'; we do not compute a Dice for each validation case and then average over all cases but pretend that there is only one validation case from which we sample patches). The reason for this is that the 'global Dice' is easy to compute during training and is still quite useful to evaluate whether a model is training at all or not. A proper validation is run at the end of the training.

TLDR: the evaluation metric is the dice score computed on a subset of patches in the validation set

**Comment by TaWald** [comment]
Thank you @evandrosks for answering this issue.
